modules/monitoring-granting-users-permission-to-monitor-user-defined-projects.adoc

```diff
@@ -13,21 +13,31 @@ You can also grant developers and other users different permissions:
 * To monitor user-defined projects.
 * To configure the components that monitor user-defined projects.
 * To configure alert routing for user-defined projects.
+* To monitor or manage alerts and silences for user-defined projects.
 
-You can grant the permissions by assigning one of the following monitoring roles:
+You can grant the permissions by assigning one of the following monitoring roles and cluster roles:
 
+.Monitoring roles
 |===
 |Role name |Description
 
-|`monitoring-rules-view` | Users with this cluster role have read access to `PrometheusRule` custom resources for a user-defined project. They can also view the alerts in the *Developer* perspective of the {product-title} web console.
+|`user-workload-monitoring-config-edit` |This role is given in the `openshift-user-workload-monitoring` project. Users with this role can edit the `user-workload-monitoring-config` `ConfigMap` object to configure Prometheus, Prometheus Operator, Alertmanager, and Thanos Ruler for user-defined workload monitoring.
 
-|`monitoring-rules-edit` | Users with this cluster role can create, modify, and delete `PrometheusRule` custom resources for a user-defined project. They can also create and silence alerts in the *Developer* perspective of the {product-title} web console.
+|`monitoring-alertmanager-api-reader` |Users with this role have read access to the Alertmanager API for monitoring of user-defined projects. They can also view alert silences in the *Developer* perspective of the {product-title} web console.
 
-|`monitoring-edit` | Users with this cluster role have the same privileges as users with the `monitoring-rules-edit` cluster role. Additionally, users can create, modify, and delete `ServiceMonitor` and `PodMonitor` resources to scrape metrics from services and pods.
+|`monitoring-alertmanager-api-writer` |Users with this role have read and write access to the Alertmanager API for monitoring of user-defined projects. They can also manage alert silences in the *Developer* perspective of the {product-title} web console.
+|===
+
+.Monitoring cluster roles
+|===
+|Cluster role name |Description
+
+|`monitoring-rules-view` |Users with this cluster role have read access to `PrometheusRule` custom resources (CRs) for user-defined projects. They can also view the alerts in the *Developer* perspective of the {product-title} web console.
```

Comment:
the cluster role can be granted for specific project(s) with RoleBinding or for all projects with ClusterRoleBinding (but the latter is probably not recommended for security concerns).

Reply:
It the latter is not recommended, I would not mention it in our docs. It is better to include only the recommended way

Reply:
This actually leads me to this question: https://github.com/openshift/openshift-docs/pull/80127#issuecomment-2288770100